Makefile

```diff
@@ -88,11 +88,17 @@ install: all
 	install -d $(DESTDIR)$(LV2DIR)/$(BUNDLE)
 	install -m755 $(BUILDDIR)$(LV2NAME)$(LIB_EXT) $(DESTDIR)$(LV2DIR)/$(BUNDLE)
 	install -m644 $(BUILDDIR)manifest.ttl $(BUILDDIR)$(LV2NAME).ttl $(DESTDIR)$(LV2DIR)/$(BUNDLE)
+	install -m644 $(BUILDDIR)$(LV2NAME).ttl $(DESTDIR)$(LV2DIR)/$(BUNDLE)
+	install -m644 default-preset.ttl modgui.ttl $(DESTDIR)$(LV2DIR)/$(BUNDLE)
+	cp -R modgui $(DESTDIR)$(LV2DIR)/$(BUNDLE)/modgui
```

Comment:
There should be a conditional. These files should only be installed when installing for the MOD.
Other x42-plugins use the MOD environment variable.

Reply:
compare to 

https://github.com/x42/fil4.lv2/blob/61a53d6938c82584e5793448054c1d391e127450/Makefile#L313-L316